integration_tests/pages/page.ts

```diff
@@ -25,6 +26,14 @@ export default abstract class Page {
     cy.get(`#${this.pageId}`).should('exist')
   }
 
+  checkCsfrTokenForFormBasedPages = (): void => {
+    cy.get('body').then(body => {
+      body.find('form').each((idx, form) => {
```

Comment:
Pages can have 0 or more forms (end journey pages have 0 forms, other pages have their own form plus optionally the Cookies form in the cookie header
For each form found on the page we need to ensure it has a `_csrf` field with a value